tests/dummy/app/controllers/class-diagram-primitives-demo.js

```diff
@@ -63,46 +77,41 @@ export default Ember.Controller.extend({
       linkComposition.setLabelText('endMultiplicity', '*');
       linkComposition.setLabelText('description', 'Composition');
 
-      var linkDependency = new joint.shapes.flexberryUml.Dependency({
+      var linkDependency = new Dependency({
         source: { x:100, y:170 },
         target: { x:300, y:170 },
       });
 
-      var linkRealization = new joint.shapes.flexberryUml.Realization({
```

Comment:
Why is it deleted and not replaced?

Reply:
The point is that joint.js does not support connections within links. I suppose we should discuss what to do.